circle.yml

```diff
@@ -57,3 +57,8 @@ jobs:
           command: |
             chmod +x /app/devops/ci/circleci_deploy.sh
             /app/devops/ci/circleci_deploy.sh
+      - flake8:
+          name: Flake8
+          command: |
+            flake8 server
```

Comment:
Not sure if the python modules are installed globally on CircleCI. We may have to run it using `env/bin/flake8`